app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -0,0 +1,85 @@
+/* global Runtime */
+
+/**
+ * Callback that dismisses the Runtime Banner
+ */
+function handleDismissRuntimeBanner() {
+  const runtimeBanner = document.querySelector('.runtime-banner');
+  if (runtimeBanner) {
+    runtimeBanner.remove();
+  }
+}
+
+/**
+ * This function redirects the browser to custom schemes, i.e. deep link into
+ * mobile apps. A separate function is needed in this case in order to test
+ * the feature using Cypress. More details can be found here:
+ * https://medium.com/cypress-io-thailand/understand-stub-spy-apis-that-will-make-your-test-better-than-ever-on-cypress-io-797cb9eb205a#b6ad
+ *
+ * @param {string} targetURI - The target custom scheme URI
+ */
+function launchCustomSchemeDeepLink(targetURI) {
+  window.location.href = targetURI;
+}
+
+/**
+ * Function that run on every page load (including InstantClick redirects) and
+ * is in charge of the setup that runs the following features:
+ * - addEventListener to dismiss the banner when users tap on X button
+ * - When the user lands in "/r/mobile"
+ *   - Timeout that presents the fallback page if couldn't seamless deep link
+ *   - Setup the correct fallback button links based on platform (iOS/Android)
+ */
+function initializeRuntimeBanner() {
+  // This will provide the dismiss functionality for the Runtime Banner
+  const bannerDismiss = document.querySelector('.runtime-banner__dismiss');
+  if (bannerDismiss) {
+    bannerDismiss.addEventListener('click', handleDismissRuntimeBanner);
+  }
+
+  // If the "Install now"/"Try again" buttons exist in the DOM it means we are
+  // trying to deep link into the mobile app after being redirected by the
+  // Runtime Banner itself (the browser is currently in `/r/mobile`)
+  const installNowButton = document.getElementById('link-to-mobile-install');
+  const retryButton = document.getElementById('link-to-mobile-install-retry');
+  if (!installNowButton || !retryButton) {
+    return;
+  }
+  // Since we were redirected to `/r/mobile` by the Banner itself we can safely
+  // dismiss it by re-using the handleDismissRuntimeBanner function
+  handleDismissRuntimeBanner();
+
+  // The target path comes in via GET request query params
+  const urlParams = new URLSearchParams(window.location.search);
+  const targetPath = urlParams.get('deep_link');
+
+  // Constants - they will become dynamic (configurable by creators) in upcoming releases
+  const FOREM_IOS_SCHEME = 'com.forem.app';
+  const FOREM_APP_STORE_URL =
+    'https://apps.apple.com/us/app/dev-community/id1439094790';
+  const FOREM_GOOGLE_PLAY_URL =
+    'https://play.google.com/store/apps/details?id=to.dev.dev_android';
```

Comment:
Love this improvement 👏🏽

Reply:
I agree 100% with you @rhymes, being able to configure these is definitely part of the plan. I shared some of that context [in this comment](https://github.com/forem/forem/pull/13190#discussion_r618335855). The tl;dr is that it's contemplated by the [feature 3](https://github.com/forem/rfcs/blob/d28ba6ce61c2cda9d4018b4237ec4866be4359b9/text/0072-runtime-banner-with-deep-linking.md#admin-panel-configuration-feature-3) section of the RFC. Coming soon after we are able to confirm all the moving parts here work properly 🤓 

The implementation details will definitely require some thought, as you mentioned these are hardcoded in the JS and those configs live in the server, but I'm optimistic we'll be able to figure out an elegant solution when tackling that challenge.